ui/src/services/router/TuringRouter.js

```diff
@@ -138,25 +138,13 @@ export class TuringRouter {
       // Copy the final response route id to the top level, as the default route
       obj.config.default_route_id =
         obj.config["ensembler"].nop_config["final_response_route_id"];
-      delete obj.config["ensembler"];
     } else if (obj.config.ensembler.type === "standard") {
       // Copy the fallback response route id to the top level, as the default route
       obj.config.default_route_id =
         obj.config["ensembler"].standard_config["fallback_response_route_id"];
-      delete obj.config["ensembler"].standard_config[
-        "fallback_response_route_id"
-      ];
     } else {
       // Docker or Pyfunc ensembler, clear the default_route_id
       delete obj.config["default_route_id"];
-      if (obj.config.ensembler.type === "pyfunc") {
-        // Delete the docker config
-        delete obj.config["ensembler"].docker_config;
-      }
-      // Docker/Pyfunc ensembler CPU limit
-      if (obj.config.ensembler.resource_request?.cpu_limit === "") {
-        delete obj.config.ensembler.resource_request.cpu_limit;
-      }
```

Comment:
This entire check was ineffective and was doing nothing since it was checking a field that doesn't exist - `obj.config.ensembler.resource_request` does not exist. The intended path should've been `obj.config.ensembler.docker_config.resource_request` or `obj.config.ensembler.pyfunc_config.resource_request`.

Reply:
The delete is done is ensembler.is instead?

Reply:
Yep that's right, because `toJSON` in `Ensembler.js` gets called after the entire return value of `toJSON` in `TuringRouter.js` gets returned.